src/server/api/controllers/messages-controller.js

```diff
@@ -78,15 +78,32 @@ const deleteMessage = async (messagesId) => {
 };
 
 const createMessage = async (body) => {
-  await knex('channel_messages').insert({
+  const newMessage = {
     message: body.message,
     created_at: moment().format(),
     fk_user_id: body.userId,
     fk_channel_id: body.channelId,
+  };
+  const messageId = await knex('channel_messages').insert(newMessage);
+  const userInCahnnel = knex
```

Comment:
> Also, you can immediately put `await` here and skip having `newRowForEachUser` at all. This will be a bit shorter then.

I did but got this error  TypeError: usersInChannel.forEach is not a function. That is way I leaved as it is.